src/MOI_wrapper.jl

```diff
@@ -346,19 +417,46 @@ end
 function MOI.get(
     optimizer::Optimizer,
     attr::MOI.ConstraintPrimal,
-    ci::MOI.ConstraintIndex,
+    ci::MOI.ConstraintIndex{MOI.VectorAffineFunction{Float64}},
 )
     MOI.check_result_index_bounds(optimizer, attr)
-    return optimizer.sol.slack[MOI.Utilities.rows(optimizer.cones, ci)]
+    return convert(
+        Vector{Float64},
+        optimizer.sol.slack[MOI.Utilities.rows(optimizer.cones_real, ci)],
+    )
+end
```

Comment:
```suggestion
end

```
Please add new lines between methods